SPECS/dwarves/dwarves.spec

```diff
@@ -0,0 +1,508 @@
+%define libname libdwarves
+%define libver 1
+Summary:        Debugging Information Manipulation Tools (pahole & friends)
+Name:           dwarves
+Version:        1.21
+Release:        4%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://acmel.wordpress.com
+Source:         http://fedorapeople.org/~acme/dwarves/%{name}-%{version}.tar.xz
+Patch1:         0001-btf-Remove-ftrace-filter.patch
+BuildRequires:  cmake >= 2.8.12
+BuildRequires:  elfutils-devel >= 0.130
+BuildRequires:  gcc
+BuildRequires:  zlib-devel
+Requires:       %{libname}%{libver} = %{version}-%{release}
+
+%description
+dwarves is a set of tools that use the debugging information inserted in
+ELF binaries by compilers such as GCC, used by well known debuggers such as
+GDB, and more recent ones such as systemtap.
+
+Utilities in the dwarves suite include pahole, that can be used to find
+alignment holes in structs and classes in languages such as C, C++, but not
+limited to these.
+
+It also extracts other information such as CPU cacheline alignment, helping
+pack those structures to achieve more cache hits.
+
+These tools can also be used to encode and read the BTF type information format
+used with the Linux kernel bpf syscall, using 'pahole -J' and 'pahole -F btf'.
+
+A diff like tool, codiff can be used to compare the effects changes in source
+code generate on the resulting binaries.
+
+Another tool is pfunct, that can be used to find all sorts of information about
+functions, inlines, decisions made by the compiler about inlining, etc.
+
+One example of pfunct usage is in the fullcircle tool, a shell that drivers
+pfunct to generate compileable code out of a .o file and then build it using
+gcc, with the same compiler flags, and then use codiff to make sure the
+original .o file and the new one generated from debug info produces the same
+debug info.
+
+Pahole also can be used to use all this type information to pretty print raw data
+according to command line directions.
+
+Headers can have its data format described from debugging info and offsets from
+it can be used to further format a number of records.
+
+The btfdiff utility compares the output of pahole from BTF and DWARF to make
+sure they produce the same results.
+
+%package -n %{libname}%{libver}
+Summary:        Debugging information  processing library
+
+%description -n %{libname}%{libver}
+Debugging information processing library.
+
+%package -n %{libname}%{libver}-devel
+Summary:        Debugging information library development files
+Requires:       %{libname}%{libver} = %{version}-%{release}
+
+%description -n %{libname}%{libver}-devel
+Debugging information processing library development files.
+
+%prep
+%setup -q
+%patch1 -p1
+
+%build
+%cmake -DCMAKE_BUILD_TYPE=Release .
+%__cmake --build . %{?_smp_mflags} --verbose
+
+%install
+rm -Rf %{buildroot}
+DESTDIR="%{buildroot}" %__cmake --install .
```

Comment:
Similarly, I believe we should have a `%cmake_install` macro to replace this whole line.